src/server/json_family_test.cc

```diff
@@ -1393,6 +1393,11 @@ TEST_F(JsonFamilyTest, Merge) {
   EXPECT_EQ(resp, "OK");
   resp = Run({"JSON.GET", "j1", "$"});
   EXPECT_EQ(resp, R"([{"a":"z","c":{"d":"e"}}])");
+
+  resp = Run({"JSON.SET", "foo", "$", "{}"});
+  EXPECT_EQ(resp, "OK");
+  resp = Run({"JSON.MERGE", "foo", "$", R"({"a":2} {"b":2})"});
+  EXPECT_EQ(resp, "OK");
```

Comment:
please also verify the result value

Reply:
yep I also verified it with `Valkey` to make sure the results consistent

Reply:
why "a" has been merged but "b" have not?

Reply:
because there is no comma, it's ignored. Same behaviour in Valkey but I wanted to leave it as is as reminder that there is no error or something

Reply:
i find it confusing because it's out of scope of this PR (json parsing).
Lets have 2 fields  `f1`, `common` in foo, 2 fields that are merged: `f2`, `common` and show how f1, f2, common appear in the final result.

Reply:
@romange done. I also tested the same command/query with Valkey and they give the same result (the only difference is the order, common is second but that doesn't change the correctness since json is unordered)